.markdownlint-cli2.yaml

```diff
@@ -1,4 +1,5 @@
 config:
-  ul-indent: false # MD007
-  line-length: false # MD013
-  no-duplicate-heading/no-duplicate-header: false # MD024
+    ul-indent: false  # MD007
+    line-length: false  # MD013
+    no-duplicate-header: false  # MD024
+    reference-links-images: false  # MD052
```

Comment:
There's no point changing this as it's managed centrally [here](https://github.com/sous-chefs/repo-management/blob/main/standardfiles/cookbook/.markdownlint-cli2.yaml)